data/mods/DinoMod/monsters/dinosaur.json

```diff
@@ -1337,5 +1337,35 @@
     "anger_triggers": [ "STALK", "PLAYER_WEAK", "HURT" ],
     "placate_triggers": [ "MEAT" ],
     "categories": [ "DINOSAUR" ]
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_kosmoceratops",
+    "name": { "str_sp": "Kosmoceratops" },
+    "copy-from": "mon_triceratops",
+    "weight": "1814 kg",
+    "melee_skill": 9,
+    "armor_cut": 7,
+    "armor_bash": 8,
+    "hp": 670,
+    "zombify_into": "mon_zosmoceratops",
+    "description": "A cousin of the Triceratops, This beast has returned from extinction, and although it's a herbivore it can do alot of damage when angered.",
+    "reproduction": { "baby_egg": "egg_kosmoceratops", "baby_count": 24, "baby_timer": 23 },
+    "special_attacks": [ [ "EAT_CROP", 24 ], { "id": "impale" }, [ "STRETCH_ATTACK", 7 ] ]
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_amargasaurus",
+    "name": { "str_sp": "Amargasaurus" },
+    "copy-from": "mon_apatosaurus",
+    "weight": "2630 kg",
+    "melee_skill": 10,
+    "armor_cut": 8,
+    "armor_bash": 9,
+    "hp": 267,
+    "zombify_into": "mon_zamargasaurus",
+    "description": "A herbivourous sauropod from the cretaceous, it seems to pose little threat to you.",
+    "reproduction": { "baby_egg": "egg_amargasaurus", "baby_count": 25, "baby_timer": 23 },
+    "special_attacks": [ [ "EAT_CROP", 27 ], { "id": "impale" }, [ "STRETCH_ATTACK", 9 ] ]
```

Comment:
```suggestion
```
Should be the same or worse than Apatosaurus unless you have a source showing those spikes were used offensively